### PR TITLE
Embed tweaks, refactor tests, graphql code splitting

### PR DESCRIPTION
**Pull request by narthur**
Closes #298

### CODE REVIEW

**Comment by narthur** [comment]
> @narthur Overall looks good, but there's a 10KB bundle impact from this PR. Is there a way to do this without impacting the bundle?

Looking into it!

**Comment by narthur** [comment]
Bug on this one I need to fix--when the video lands in the miniplayer, it pauses.

File: src/components/organisms/navigation.tsx [hunk 1]
```diff
@@ -93,7 +94,9 @@ const Navigation = ({
 							<LanguageButton
 								buttonType="secondary"
 								onClick={(baseUrl) => {
-									router.push(`/${baseUrl}/`);
+									router.push(`/${baseUrl}/`).then(() => {
+										// noop
```
Comment (mattleff) [inline src/components/organisms/navigation.tsx:98]: Is this needed?

File: src/containers/sponsor/conferences.tsx [hunk 2]
```diff
@@ -25,9 +22,11 @@ function SponsorConferences({
 	nodes,
 	data: { sponsor },
 	pagination,
-}: Must<SponsorConferencesProps> & {
-	data: { sponsor: Must<SponsorPivotFragment> };
-}): JSX.Element {
+}: Must<SponsorConferencesProps>): JSX.Element {
+	if (!sponsor) {
```
Comment (mattleff) [inline src/containers/sponsor/conferences.tsx:26]: Do we need this change?

File: src/containers/sponsor/series.tsx [hunk 3]
```diff
@@ -25,9 +22,11 @@ function SponsorSeries({
 	nodes,
 	data: { sponsor },
 	pagination,
-}: Must<SponsorSeriesProps> & {
-	data: { sponsor: Must<SponsorPivotFragment> };
-}): JSX.Element {
+}: Must<SponsorSeriesProps>): JSX.Element {
+	if (!sponsor) {
```
Comment (mattleff) [inline src/containers/sponsor/series.tsx:26]: Do we need this change? There are several of these scattered around...

File: src/lib/makeQueryClient.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+			queries: {
+				refetchOnWindowFocus: false,
+				// WORKAROUND: https://react-query.tanstack.com/guides/testing#set-cachetime-to-infinity-with-jest
+				cacheTime: IS_TEST ? Infinity : undefined,
+				staleTime: IS_TEST ? Infinity : undefined,
+			},
+		},
+	});
+
+	if (DEBUG) {
```
Comment (mattleff) [inline src/lib/makeQueryClient.ts:18]: How will this ever be true?

File: src/lib/useIntlMessages.ts [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+		async function loadMessages() {
+			const messages = await getIntlMessages(language);
+			if (!active) return;
+			setMessages(messages);
+			setLoading(false);
+		}
+
+		loadMessages().then(() => {
+			// noop
```
Comment (mattleff) [inline src/lib/useIntlMessages.ts:27]: Is this needed?

**Review by mattleff — APPROVED** [review]
👍🏻 This would be easier to review if it was 5 or 6 PRs, but it's probably ok.

File: src/containers/sponsor/conferences.tsx (hunk 2 above)
Comment (mattleff) [reply src/containers/sponsor/conferences.tsx]: Fixed 72f3d61.

File: src/containers/sponsor/series.tsx (hunk 3 above)
Comment (mattleff) [reply src/containers/sponsor/series.tsx]: Fixed: 72f3d61.

**Pull request closed without merging**